docs/eventing/channels/README.md

```diff
@@ -10,24 +10,29 @@ procedure:
     ```
 -->
 
-Channels are Kubernetes Custom Resources which define a single event forwarding
-and persistence layer. Messaging implementations may provide implementations of
-Channels via the
+Channels are Kubernetes Custom Resources which define a single event forwarding and persistence layer.
+Messaging implementations may provide implementations of Channels via the
 [ClusterChannelProvisioner](https://github.com/knative/eventing/blob/master/pkg/apis/eventing/v1alpha1/cluster_channel_provisioner_types.go#L35)
-object, supporting different technologies, such as Apache Kafka or NATS
-Streaming.
+object, supporting different technologies, such as Apache Kafka or NATS Streaming.
```

Comment:
Maybe add saying that CCP is deprecated and something that:
may provide implementes of channels via the CRD (Operator pattern) or (deprecated) ClusterChannelProvisioner...

or something like that?